### ISSUE TITLE
Error when calling Irr for permutation group

### ISSUE THREAD

**Issue by ThomasBreuer** [gap-system/gap#1515]
The following happens in GAP 4.8.7 as well as in the current master branch,
without packages.
```
gap> GAPInfo.Version;
"4.8.7"
gap> s10:= SymmetricGroup( 10 );;
gap> Irr( Stabilizer( s10, [ 1 .. 3 ], OnSets ) );; # this works
gap> Irr( Stabilizer( s10, [ 2 .. 4 ], OnSets ) );; # this does not work
Error, List Element: <list>[2] must have an assigned value in
  img := opr( img, S.transimages[bimg[1]] )
 ; at /users/sam/gap/gap4r8/lib/ghomperm.gi:108 called from 
ImageSiftedBaseImage( S, OnTuples( BaseStabChain( S ), elm ), S.idimage, 
 OnRight ) at /users/sam/gap/gap4r8/lib/ghomperm.gi:1008 called from
ImagesRepresentative( InverseGeneralMapping( hom ), elm 
 ) at /users/sam/gap/gap4r8/lib/ghomperm.gi:1101 called from
PreImagesRepresentative( hom, conj 
 ) at /users/sam/gap/gap4r8/lib/clashom.gi:2786 called from
TFCanonicalClassRepresentative( D.group, [ el ] 
 ) at /users/sam/gap/gap4r8/lib/ctblperm.gi:62 called from
D.identification( D, el 
 ) at /users/sam/gap/gap4r8/lib/ctblgrp.gi:1647 called from
...  at line 4 of *stdin*
you can 'return;' after assigning a value
brk> 
```

**Comment by markuspf** [comment]
This is fixed in #1390 by @hulpke 

Maybe we need to backport that fix?

**Comment by alex-konovalov** [comment]
@ThomasBreuer thank you for reporting this - I've backported the fix from the master branch in https://github.com/gap-system/gap/pull/1566 so this will be fixed in GAP 4.8.8 coming shortly.
